### PR TITLE
Add How to review a Pull Request article

### PR DESCRIPTION
**Pull request by MariagraziaAlastra**
Added an article containing a checklist for when reviewing PRs.

### CODE REVIEW

File: programming-mission/pr-review.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
+# How to review a Pull Request
+
+This is a checklist intended for Language Leaders and Community Leaders.
+Here you'll find everything that should be addressed when reviewing a Pull Request.
+Before merging, make sure the latest changes have been deployed to beta and throughly tested; otherwise, or in case you're unsure or have issues, please reach out to either **@moollaza** or **@maria** on Slack.
+
+## General
+- Is the PR related to an existent GitHub issue?
```
Comment (russellholt) [inline programming-mission/pr-review.md:9]: Is this PR related to a GitHub Issue?

File: programming-mission/pr-review.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
+# How to review a Pull Request
+
+This is a checklist intended for Language Leaders and Community Leaders.
+Here you'll find everything that should be addressed when reviewing a Pull Request.
+Before merging, make sure the latest changes have been deployed to beta and throughly tested; otherwise, or in case you're unsure or have issues, please reach out to either **@moollaza** or **@maria** on Slack.
+
+## General
+- Is the PR related to an existent GitHub issue? 
+    - If not, and if it's a new IA, the contributor should [open a new project topic on the forum](). The PR will be on hold until the idea is approved.
```
Comment (russellholt) [inline programming-mission/pr-review.md]: project topic?

File: programming-mission/pr-review.md (hunk 1 above)
Comment (MariagraziaAlastra) [reply programming-mission/pr-review.md:9]: We're talking about Spices, Goodies and Fatheads here

File: programming-mission/pr-review.md (hunk 2 above)
Comment (MariagraziaAlastra) [reply programming-mission/pr-review.md]: Edited

**Comment by moollaza** [comment]
Made some minor changes, overall this looks like a good start.

@duckduckgo/community-leaders @duckduckgo/language-leaders -- any thoughts on this? is it helpful? Do you have any unanswered questions?

**Comment by gautamkrishnar** [comment]
Looking great 👍

File: programming-mission/pr-review.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+- Will any code break if the API response is undefined?
+    - Make sure objects exist before accessing nested properties
+    - You can use `DDG.getProperty()` to get properties which may undefined (e.g. `DDG.getProperty(my_obj, 'some.property.name')`)
+
+#### CSS:
+- Is there any custom CSS?
+    - Custom CSS should be kept to a minimum. We prefer none, if possible
+- Are all CSS rules name-spaced?
+    - None of the CSS should be able to affect other Instant Answers or page elements
+- Font-colour, font-size and font-family should be defined by template variants, or our DDG classes (e.g. `.tx--14`, `.bg-clr--grey`, `.tx-clr--green`)
```
Comment (rasikapohankar) [inline programming-mission/pr-review.md:71]: Maybe also include a link to the [styleguide](https://duckduckgo.com/styleguide)?

File: programming-mission/pr-review.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+    - Regexes should only be used when matching specific patterns or types of characters (numbers, unicode symbols)
+- Trigger words must be lowercase
+
+#### Cheat Sheets
+- Make sure the description is not redundant
+- Is the spelling correct?
+- "Title Case" should be used for name and sections
+- "Sentence case" should be used for description and explanations
+- Ensure the `CheatSheet.t` test passes
+- Use JSONLint to check the file and the text format
```
Comment (sahildua2305) [inline programming-mission/pr-review.md:94]: @MariagraziaAlastra @moollaza [This](https://duckduckgo.com/?q=json+validate&t=h_&ia=answer) instead of some random JSONLint? 😉
